### DISCUSSION TITLE
Asset Overridement

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#399] [Q&A] [answered]
I'm having an issue (specifically with .json files) where the assets in the Bin folder aren't replaced when I try to save over them. I swear I've saved this file like 3 times and it's still the version from 2 days ago. In fact, I'm sure of it; the visibility of strumlines confirms it
![image](https://github.com/user-attachments/assets/79cf19b7-934d-4949-a7c0-7b7dacd36012)
This also applies to files that I've deleted. Files that I've deleted in the Assets folder aren't gone in the Bin/assets folder

**Comment by THEkatinamicrowave** [comment]
I might try to learn how command prompt works and make my own custom command that:

1. Deletes the bin assets folder
2. Then compile the stuff

**Comment by moxie-coder** [comment]
> I'm having an issue (specifically with .json files) where the assets in the Bin folder aren't replaced when I try to save over them. I swear I've saved this file like 3 times and it's still the version from 2 days ago. In fact, I'm sure of it; the visibility of strumlines confirms it ![image](https://private-user-images.githubusercontent.com/89645987/373974039-79cf19b7-934d-4949-a7c0-7b7dacd36012.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyNTczNzAsIm5iZiI6MTcyODI1NzA3MCwicGF0aCI6Ii84OTY0NTk4Ny8zNzM5NzQwMzktNzljZjE5YjctOTM0ZC00OTQ5LWE3YzAtN2I3ZGFjZDM2MDEyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA2VDIzMjQzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI2MDBhODk0ZGM1NDdkNDllYzFiNWVjZjY4YmM0ODA2YmNiZmVmZTZmZWQyMGNhNmFlYjlkZTc0ZDJlOTliMGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7GwFeHamJiZbHsjlXfSFUtJIQ23MbwKSaLos2s4dqk0) This also applies to files that I've deleted. Files that I've deleted in the Assets folder aren't gone in the Bin/assets folder

you’re not supposed to put them in the assets folder

**Reply by THEkatinamicrowave** [reply]
Assets folder extended. I'm talking about (for example) deleting a file in `assets/songs/bopeebo` and then it still being there in `bin/assets/songs/bopeebo` post-compile

**Reply by r6915ee** [reply, marked as answer]
This usually only happens if you didn't delete the export folder before compiling. Lime usually keeps exported assets that have been deleted in the source code repository.

**Reply by THEkatinamicrowave** [reply]
yeah I'm just kinda resorting to deleting the assets and manifest folders whenever I recompile and it seems to be working
